stellar-contract-env-host/src/host.rs

```diff
@@ -351,7 +367,25 @@ impl Env for Host {
     }
 
     fn vec_get(&mut self, v: RawVal, i: RawVal) -> RawVal {
-        todo!()
+        let index = if u32::is_val_type(i) {
+            unsafe { u32::unchecked_from_val(i) }
+        } else {
+            panic!("index has wrong type, must be u32")
+        };
```

Comment:
I think if you're going to do this in a checked way (you're checking with is_val_type, you may as well use:
```suggestion
        let index: u32 = i.try_into().expect("index has wrong type, must be u32");
```

Reply:
That's a great suggestion, fixed!